spec/index.bs

```diff
@@ -569,6 +561,63 @@ enum GPUBindingType {
 };
 </script>
 
+  * {{GPUBindGroupLayoutBinding/textureDimension}}, {{GPUBindGroupLayoutBinding/multisampled}}:
+    The dimensions and multi-sampling properties of texture bindings.
+
+    Note: This allows Metal-based implementations to back the respective bind
+    groups with `MTLArgumentBuffer` objects that are more efficient to bind at
+    run-time.
+
+  * {{GPUBindGroupLayoutBinding/dynamic}}:
+    For [=uniform-buffer=], [=storage-buffer=], and [=readonly-storage-buffer=] bindings, 
+    indicates that the binding has a dynamic offset. One offset must be passed to 
+    setBindGroup for each dynamic binding in increasing order of
+    {{GPUBindGroupLayoutBinding/binding}} number.
+
+### {{GPUDevice/createBindGroupLayout()|GPUDevice.createBindGroupLayout(GPUBindGroupLayoutDescriptor)}} ### {#GPUDevice-createBindGroupLayout}
+
+A valid {{GPUBindGroupLayout}} object has an <dfn>internal map</dfn> of |u32| <dfn>binding number</dfn>s to internal <dfn>layout binding</dfn> objects. 
```

Comment:
This means that the binding numbers can be arbitrary. That sounds good but our implementation currently requires bindings numbers between 0 and 15 and would likely introduce a map in the BGL from application-facing binding numbers to packed bindings numbers in 0..N.

Reply:
Can you clarify if you are going to have a problem with arbitrary binding numbers or not? Our implementation works, and I could think of a case where the users would want the numbers to exceed 15.

Reply:
We aren't going to have problems with arbitrary numbers, sorry for the confusion. I just wanted to make sure this was done being aware that an alternative was to require numbers in a small range corresponding to maxBindingsPerGroup.

Reply:
@Kangz Do you mean you'd like this text to change at all, or are you just pointing out the work you have to do on your end?

Reply:
Allowing for arbitrary indices is easier for developers and trivial to reorder inside the implementation.

Reply:
@JusSn I'm pointing out an alternative that I would like considered if at least for a couple seconds. @litherum just did and I don't have a strong opinion on the subject so being nice to developers sounds good.